app/views/RoomInfoView/index.js

```diff
@@ -214,7 +214,7 @@ class RoomInfoView extends React.Component {
 		}
 
 		const permissions = await RocketChat.hasPermission([editRoomPermission], room.rid);
-		if (permissions[0] && !room.prid) {
+		if (permissions[0]) {
```

Comment:
Can you write a simple e2e test for this?

Check this test: https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/e2e/tests/room/04-discussion.spec.js#L134

Reply:
Sure, I can!